svc-api/main.go

```diff
@@ -48,6 +48,21 @@ func main() {
 		}
 		basicAuth := r.Header.Get("Authorization")
 		var basicAuthToken string
+		// This if block added to fix issue odim-98
+		if basicAuth != "" {
+			var URLs_not_requiring_basic_auth = []string {"/redfish/v1","/redfish/v1/SessionService"}
+			var found bool = false
+			for _, item := range URLs_not_requiring_basic_auth {
+				if item == path {
+					found = true
+					break
+				}
+			}
+			if found {
```

Comment:
This if condition is not required. body of this can be replaced at line 57

Reply:
Done.